### DISCUSSION TITLE
my request is approved but benefits not added yet its been 3 days

### DISCUSSION THREAD

**Discussion by devsonix1** [community/community#140248] [GitHub Education]
### Hi everyone,

![Screenshot 2024-09-30 143750](https://github.com/user-attachments/assets/31a4f62d-4710-42f0-95a8-be303845ebb3)

**Comment by InboraStudio** [comment]
We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
We will be locking this discussion. If you haven't received your benefits after approval within 5 days please raise a new discussion so we can address it 👍
